package.json

```diff
@@ -42,6 +42,7 @@
   "devDependencies": {
     "@types/mocha": "^8.2.2",
     "@types/node": "^15.12.2",
+    "@types/graphql": "^14.5.0",
```

Comment:
Why do we need these types if we remove `graphql` from the dep?
![image](https://user-images.githubusercontent.com/477945/123634875-43e91f00-d81b-11eb-83c3-f5638bee74a4.png)

Reply:
I don't know for sure yet. I think it's related to the `graphql-tag` dependency. I could not get the github actions to run successfully without them. I'm currently investigating.